app/models/holiday.rb

```diff
@@ -0,0 +1,14 @@
+class Holiday < ApplicationRecord
+  belongs_to :user, required: false
+
+  def self.merged_for_calendar_view
+    select(<<-SQL
+      DISTINCT ON(holidays.start_at, holidays.end_at, holidays.title)
+        holidays.title, holidays.start_at, holidays.end_at, holidays.title || COALESCE(' - ' || string_agg(users.name, ', '), '') AS title
+    SQL
+          )
+      .joins('LEFT JOIN users ON users.id = holidays.user_id')
```

Comment:
I think `includes(:users)` would do the same here, plus the users would then be eager-loaded too as a bonus.